### PR TITLE
GraalVM Integration blog post

### PR DESCRIPTION
**Pull request by ilopmar**
@kroegerk please review this. I'm pretty sure the English can be improved a lot in some parts of the post. I did my best :-)
I've set the date on October 12th, next Monday. Feel free to change it to other day if you want.

@JasonTypesCodes I would like to get your input about this. I've explained the technical stuff but I don't really know if I went to far or if I need to take a step back and simplify a few things. Please let me know what you think about it.

### CODE REVIEW

File: posts/2020-10-12-test-graalvm.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+Over this time a lot of things have changed, and those initial two applications have become 26 test applications with 39
+different jobs. This is because some of them test the same thing but for different options. For example in the 
+[Micronaut JDBC](https://github.com/micronaut-graal-tests/micronaut-data-jdbc-graal) test application, there are branches
+for the all the databases supported: H2, Postgres, Oracle, MariaDB, SQL Server and MySQL.
+
+Also, the number of jobs have been multiplied by four because we now test GraalVM for JDK 8 and JDK11, and we also test
+ at two Micronaut versions (at this moment `2.0.x` and `2.1.x`). This is how the CI pipeline looks like today:
+
+
+![CI pipeline today](2020-10-12-ci-pipeline-today.png)
```
Comment (JasonTypesCodes) [inline posts/2020-10-12-test-graalvm.md:76]: This image is really large.   Is there a way we can get this across without such a large image?

**Comment by JasonTypesCodes** [comment]
Excellent work @ilopmar !  I've made a number of suggestions here: https://github.com/micronaut-projects/static-website/pull/176/commits/3b3ef994b5dbf6173a6a1073fec6ae0733a02aa2  Feel take or leave any of them.

File: posts/2020-10-12-test-graalvm.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+title: How do we test Micronaut and GraalVM integration?
```
Comment (JasonTypesCodes) [inline posts/2020-10-12-test-graalvm.md:2]: I feel like there is a better title for this but I haven't come up with it yet...

File: posts/2020-10-12-test-graalvm.md (hunk 1 above)
Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md:70]: Yeah, I wasn't sure about it either. I've pushed a new version moving the JDK11 tests to a new column on the right. Now everything fits in the screen without doing any scroll:

image

File: posts/2020-10-12-test-graalvm.md (hunk 2 above)
Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md:2]: I'm not happy with it, but not really sure about other alternatives. Maybe @kroegerk can help us :)

**Comment by ilopmar** [comment]
> Excellent work @ilopmar ! I've made a number of suggestions here: [3b3ef99](https://github.com/micronaut-projects/static-website/commit/3b3ef994b5dbf6173a6a1073fec6ae0733a02aa2) Feel take or leave any of them.

Thanks! I'll keep you suggestions, your English is way better than mine.

I promise that some day I'll put all the commas before the `and`, but as in Spanish is not the same, I always forget it... :blush:

File: posts/2020-10-12-test-graalvm.md (hunk 2 above)
Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:2]: @ilopmar @JasonTypesCodes 

Suggestions:

Testing Micronaut and GraalVM Integration
A Simple Guide to Testing Micronaut and GraalVM Integration
Testing Micronaut and GraalVM Integration: A Step-by-Step Guide
Testing Micronaut and GraalVM Integration Step-by-Step
Tips for Testing Micronaut and GraalVM Integration
Testing Tips: Micronaut and GraalVM Integration
Testing Micronaut and GraalVM Integration the Easy Way
The Easiest Way to Test Micronaut and GraalVM Integration
Test Your Micronaut and GraalVM Integration with Ease
How to Test Micronaut and GraalVM Integration

Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md:2]: :exploding_head:

File: posts/2020-10-12-test-graalvm.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+date: Oct 12, 2020
+author: Iván López
+image: 2020-10-12.jpg
```
Comment (ilopmar) [inline posts/2020-10-12-test-graalvm.md:5]: @kroegerk you haven't added the image in this PR

File: posts/2020-10-12-test-graalvm.md (hunk 2 above)
Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:2]: Now that I've read over the article more closely, I see that it's not really a tutorial for users, but an explanation of how the Micronaut team tests integration. Some of my suggestions above are not relevant in that case. I'll provide a new list momentarily.

Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:2]: New Suggestions:

Testing Micronaut and GraalVM Integration
How Micronaut and GraalVM Integration Is Tested
Micronaut GraalVM Integration: Continual Improvements FTW
Improving Micronaut and GraalVM Integration
Micronaut and GraalVM Integration: The Process and the Results
How We Improved Micronaut GraalVM Integration and What That Means for You
Micronaut GraalVM Integration Yesterday and Today
Improvements to Micronaut GraalVM Integration
Micronaut and GraalVM Integration: Better than Ever
Micronaut GraalVM Integration: A Peek at the Development Process
Micronaut and GraalVM Integration: A Success Story
Micronaut and GraalVM: An Integration Story
Micronaut and GraalVM: An Integration Story and a Happy Ending
Integrating Micronaut and GraalVM: Lessons Learned
Micronaut GraalVM Integration: What We've Learned and How We've Improved the Process
Making Micronaut and GraalVM Play Well Together

File: posts/2020-10-12-test-graalvm.md (hunk 3 above)
Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:5]: Oh, dear. I'm sorry @ilopmar. I assumed everything would stay together. Let me see if I can figure out how to add the image.

Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:5]: Oh! You were referring to the featured image that appears on the index page. My apologies. I usually create that file and upload it directly to the folder.

Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md:5]: Here is the featured image I made for the post. It is already in the bgimages folder.

Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md:5]: I see that you've added it to `master` branch. I've merged `master` into this branch and it's now included.

File: posts/2020-10-12-test-graalvm.md (hunk 2 above)
Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md:2]: I like (in no particular order):
- How We Improved Micronaut GraalVM Integration and What That Means for You
- Micronaut GraalVM Integration: A Peek at the Development Process
- Micronaut and GraalVM: An Integration Story and a Happy Ending

@JasonTypesCodes what do you think?

BTW @kroegerk it's amazing how many different options you can think in a few minutes. Next time I need a title for a talk I know who to ask... :rofl: :laughing:

Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md]: I've updated the title.

Comment (kroegerk) [reply posts/2020-10-12-test-graalvm.md]: Awww, shucks. Thanks. 

And @ilopmar , I'm always happy to review and make suggestions for anything you write. You know how code just makes sense and "clicks" in your brain? That's how words are in my brain! Please don't hesitate to ask – believe it or not, that kind of thing is fun for me!

**Review by kroegerk — APPROVED** [review]
Looks great!

File: posts/2020-10-12-test-graalvm.md (hunk 2 above)
Comment (ilopmar) [reply posts/2020-10-12-test-graalvm.md]: :heart_eyes:

**Pull request merged**